assessment.md

```diff
@@ -56,14 +56,68 @@ Project Management is Covered by Team 2
 * Software architecture
+  * :disappointed: No carefully thought-out contribution to the layout of the Project
+  * :confused: Very little amount of shape to the architecture to the Project
+  * :neutral_face: There is some sort of plan for the system
+  * :smiley: There is a well thought-out plan for the system
+  * :sunglasses: There is a very detailed and complete plan laid out for the system
 * Software design
+  * :disappointed: No effort to make a design
+  * :confused: Small effort made to put in place a layout of a design system
+  * :neutral_face: Some small designs are able to be seen so a solution is visible
+  * :smiley: A solution that is rich in coding layout and architectural design
+  * :sunglasses: All expectations have been exceeded with a well thought-out high
+  level software solution with both extremely detailed algorithmic and architectural
+  designs.
 * Software documentation
+  * :disappointed: No written communication of how the software works.
+  * :confused: Very little or unclear comments in the code
+  * :neutral_face: Software documentation has a decent bit of detail of the program
+  * :smiley: Detailed documentation within the project with ample details of the
+  program
+  * :sunglasses: Very detailed explanation of how to use the software with reasons
+  of why certain methods used.
 * Programming styles
+  * :disappointed: No specific style is expressed
+  * :confused: programming style is not clear
+  * :neutral_face: Some programming styles are shown
+  * :smiley: Most of program has clear styles being displayed
+  * :sunglasses: Majority of program has styles being displayed
 * Managing software complexity
+  * :disappointed: No complexity managements techniques utilized
+  * :confused: Code is frequently non-obvious and complex
+  * :neutral_face: Some basic code management is used such as simplicity or simplification
+  * :smiley: Most of code is managed using complexity reduction techniques like abstraction
+  * :sunglasses: Majority of code effectively uses complexity reduction techniques
+  such as modular design
 * Mitigating software risk
+  * :disappointed: No risk identified
+  * :confused: Minimal risk identified
+  * :neutral_face: Some risk identified and basic risk mitigation techniques used
+  * :smiley: Most risk identified and some risk mitigation techniques used
+  * :sunglasses: Majority risk identified and mitigation techniques used
 * Software testing
+  * :disappointed: No softwares tested
+  * :confused: Minimal softwares tested
+  * :neutral_face: Some softwares tested and basic recordings of quality
+  * :smiley: Most softwares tested and some recordings of overall quality
+  * :sunglasses: Majority softwares tested and recordings of overall quality
 * Software refactoring
+  * :disappointed: No software refactoring takes place
+  * :confused: Minimal changes are made to software systems
+  * :neutral_face: Some changes to software systems and some internal improvement
```

Comment:
Refactor does not necessarily mean change, especially in terms of the external behavior of a function. It's more like restructuring/optimizing the existing code. More research needs to be done to establish a better standard of code refactoring